### PR TITLE
remove prefix 'W/"' from _rev

### PR DESCRIPTION
**Pull request by nickpell**
To address https://sentry.io/dimagi/commcarehq/issues/605853916/

### CODE REVIEW

File: couchdbkit/client.py [hunk 1]
```diff
@@ -530,7 +530,7 @@ def get_rev(self, docid):
             if e.response.status_code == 404:
                 raise ResourceNotFound
             raise
-        return response.headers['ETag'].strip('"')
+        return response.headers['ETag'].strip('"').strip('W/"')
```
Comment (emord) [inline couchdbkit/client.py]: Where is the `W/"` coming from?

Comment (nickpell) [reply couchdbkit/client.py]: Never figured out exactly where.  I assume that the cloudant api sends a slightly different request.

**Comment by snopoke** [comment]
Interesting that couch returns a weak etag

**Comment by emord** [comment]
@snopoke yeah. it's weird and looks like it only happens on our self hosted instances. I quickly looked through the couch docs, but didn't see anywhere that that could be changed

**Pull request merged**